File: python/7 kyu/117.py
# https://www.codewars.com/kata/5f55ecd770692e001484af7d/train/python

# Passed

def mirror(data: list) -> list:
    if not data:
        return data

    sorted_data = sorted(data)
    result = sorted_data[::-1]
    result = sorted_data[:-1] + result

    return result

output = mirror([-3, 15, 8, -1, 7, -1])
print(output)

File: nodejs/8 kyu/117.js
// https://www.codewars.com/kata/55ecd718f46fba02e5000029/train/javascript

// Passed

function between(a, b) {
  let result = [];
  for (let i = a; i <= b; i++) {
    result.push(i);
  }

  return result;
}

const output = between(1, 4);
console.log(output);